bin/test_travis.sh

```diff
@@ -119,9 +119,15 @@ fi
 if [[ "${TEST_SYMPY}" == "true" ]]; then
     cat << EOF | python
 print('Testing SYMPY, split ${SPLIT}')
-import sympy
-if not sympy.test(split='${SPLIT}'):
+import sympy, coverage
+cov = coverage.Coverage()
+cov.start()
+if not sympy.test(split='${SPLIT}', subprocess=False):
```

Comment:
See [here](https://travis-ci.org/sympy/sympy/jobs/187108154#L3762) (side note, the output is bugged; it says "off" but it should say "enabled with unknown seed" or something like that).

So in other words, this is bad. If coverage really requires the same process, you'll need to add support for it directly in `runtests.py` so that it can work even when run in a subprocess.